### DISCUSSION TITLE
Possible to "reinstall" remotely without IPMI to use ISO?

### DISCUSSION THREAD

**Discussion by claudiopolis** [Security-Onion-Solutions/securityonion#9337] [Unsupported Versions] [answered]
I have a host running SO to monitor my home/lab network in a remote location, but it doesn't have remote console access. It was running 2.3.40 from an ISO install back in April that I had not run SOUP on. After moving house, I finally reconnected the hardware in October before I left the location thinking I could update it remotely. 

I'm  unable to get past 2.3.70 with SOUP (following the message to use "sudo BRANCH=2.3.130-20220607 soup" to first upgrade to 2.3.130), getting a highstate already enabled message. Re-running setup fails also. 

Searching error messages here and via Google, it seems the path of least resistance is to reinstall from an ISO, but I can't get back onsite until mid 2023 at the earliest, and since network setup fails, the machine is currently unusable. 

Is there any hope in trying to uninstall all of the SO packages to restore the machine to (as close as possible) a clean CENTOS install and then follow the procedure to do a network install?

TIA for any advice, tips, etc.

**Comment by dougburks** [comment, marked as answer]
In theory, you should be able to manually remove all of the configuration and get back to a relatively clean CentOS install. It's not something that we normally do, so we won't be able to provide any further assistance.

**Reply by claudiopolis** [reply]
Thanks, Doug. I wound up succeeding at just that on the 2nd attempt. 

Leaving some breadcrumbs for others that might be in a similar pickle, here is the list of steps I followed (but may be out of order, relying on scribbled notes and shell history). 

Use the so-<service>-<verb> commands to stop all SO services. 

Use the following commands to remove all docker containers and images:
```
docker kill $(docker ps -q)
docker rm $(docker ps -a -q)
docker rmi -f $(docker images -q)
```

Remove all packages from the added repos (including docker), remove all folders from /nsm and from /opt.

```
rm -Rf /var/cache/salt/
rm -Rf /var/cache/yum/x86_64/7/securityonion
rm -Rf /var/so-launcher/
rm -Rf /tmp/sogh/*
```

Find and remove all file related to nginx and all directories named securityonion (or SecurityOnion).

After that, regular network install instructions worked fine. 

Thanks, as always, to you and all the contributors for making such an AMAZING distro!!!

**Comment by M-F-P** [comment]
I just did this earlier this afternoon using Intel AMT/vPro. Establishing the initial connection that supported mounting the iso via IDER was shaky and probably wouldn't have been possible if the system wasn't a couple hundred feet away, but once everything was set up, it was slick as s**t.  

I use Mesh Commander as my vPro client.
